server/dummyData.js

```diff
@@ -1,10 +1,87 @@
 import TestRun from './models/testRun';
 const resultHelper = require('./controllers/helpers/resultHelper');
+const faker = require('faker/locale/en');
+const sample = require('lodash.sample');
+
+// Define dataset restrictions
+const frameworks = ['Cypress', 'Mamba', 'Jest'];
+const results = ['success', 'failure'];
```

Comment:
'skipped' will be a result going forward, though I suppose we can add that here when we get to that stage